sdk/src/azure/platform/az_curl.c

```diff
@@ -305,8 +305,7 @@ static AZ_NODISCARD az_result _az_http_client_curl_send_get_request(CURL* ref_cu
 /**
  * handles DELETE request
  */
-static AZ_NODISCARD az_result
-_az_http_client_curl_send_delete_request(CURL* ref_curl)
+static AZ_NODISCARD az_result _az_http_client_curl_send_delete_request(CURL* ref_curl)
```

Comment:
Fyi @antkmsft - see example of potential merge conflict introduced because of unrelated change in https://github.com/Azure/azure-sdk-for-c/pull/988 :)